automatic/thunderbird/tools/chocolateyInstall.ps1

```diff
@@ -17,11 +17,15 @@ if ($alreadyInstalled -and ($env:ChocolateyForce -ne $true)) {
   return
 }
 
-$tbProcess = Get-Process thunderbird -ea 0
-if ($tbProcess) {
-  Write-Host "Stopping running thunderbird process"
-  Stop-Process $tbProcess
-  $tbProcess = $tbProcess.Path
+$pp = Get-PackageParameters
+
+if (!$pp.NoRestart) {
+  $tbProcess = Get-Process thunderbird -ea 0
+  if ($tbProcess) {
+    Write-Host "Stopping running thunderbird process"
+    Stop-Process $tbProcess
+    $tbProcess = $tbProcess.Path
+  }
```

Comment:
Could we flip this to instead check if the process exists, if it does exist but the NoRestart is specified then output a warning about it not being stopped before installation?